TrackOverview.md

```diff
@@ -48,16 +48,18 @@ More info can be found below on how to get started.
  
 ### Entity Types
 **What?**
-* TBD
+* Entity type definitions are a mapping between the telemetry ingested to New Relic and the entities (objects) that users can interact with in New Relic UI. If you have telemetry from any source that is not supported out of the box in New Relic, you can propose a mapping to light up the New Relic experience.  
```

Comment:
```suggestion
* Entity type definitions are a mapping between the telemetry ingested to New Relic and the entities (objects) that users can interact with in New Relic UI. If you have telemetry from any source that is not supported out of the box in New Relic, you can propose a mapping to ingest data into the New Relic experience.  
```

Reply:
the term "light up" may not be clear to all users, so I'd suggest just expressing what the user can do with entity type definitions.

Reply:
the mapping enables the experiences for this entity type: Navigator, entities list view, entity overview, Lookout... I'm not sure of a better wording.

Reply:
Oh I see. so how about combining the language? to `ingest data into and light up` this way it's clear that the data ingest enables the UI to light up. 🤔

Reply:
I would prefer to keep the original wording.

Reply:
👍